x/qgb/keeper/keeper_valset.go

```diff
@@ -192,25 +186,3 @@ func (k Keeper) GetLastValsetBeforeNonce(ctx sdk.Context, nonce uint64) (*types.
 		fmt.Sprintf("couldn't find valset before nonce %d", nonce),
 	)
 }
-
-// TODO add query for this method and make the orchestrator Querier use it.
-// GetValsetByNonce returns the stored valset associated with the provided nonce.
-// Returns (nil, false, nil) if not found.
-func (k Keeper) GetValsetByNonce(ctx sdk.Context, nonce uint64) (*types.Valset, bool, error) {
-	at, found, err := k.GetAttestationByNonce(ctx, nonce)
```

Comment:
I guess this isn't needed any longer?

Reply:
yes, it was meant to be used by the orchestrator. But we have this logic on orchestrator, so no need to have it in here